docs/rsdl/rapid-rsdl-abnf.md

```diff
@@ -117,18 +135,185 @@ annotationValue      = %s"true"
 ```
 
+### Model Capabilities
+
+```ABNF
+primitivePropertyCapability = "filterable" [ filterOptions ] / "orderable" [ orderByOptions ]
+
+primitivePropertyCapabilities = "{" [ primitivePropertyCapability *( separator primitivePropertyCapability )] "}"
+
+collectionPropertyCapabilities = collectionCapabilities
+
+singleReferenceCapability = ("READ" / "UPDATE" / "REPLACE") ["{" *refCapability "}"]
+
+singleReferenceCapabilities = "{" [ singleReferenceCapability *( separator singleReferenceCapability )] "}"
+
+singleNullableReferenceCapability = singleReferenceCapability / "DELETE" noOptions
+
+singleNullableReferenceCapabilities = "{" [ singleNullableReferenceCapability *( separator singleNullableReferenceCapability )] "}"
+
+collectionReferenceCapability =  "DELETE" OWS noOptions /
+                                   "LIST" ["{" *(collectionRefCapability) "}"] /
+                                   ("READ" / "CREATE" / "REPLACE" / "UPDATE") ["{" *(refCapability) "}"]
+
+collectionReferenceCapabilities = "{" [ collectionReferenceCapability *( separator collectionReferenceCapability )] "}"
+
+```
+
+### Paths
+
+```ABNF
+paths = %s"paths" OWS ":" OWS "{" *( OWS path ) OWS "}"
+
+path =   "/" entitySet [ "/" castSegment ] [ RWS collectionRefPathCapabilities ]
+       / "/" singleton [ "/" castSegment] [ RWS singleRefPathCapabilities ]
+       / "/" serviceMember *("/" interimSegment) "/" lastSegment
+       / serviceOperationPath
+
+interimSegment = singleValuedSegment / nullableValuedSegment / castSegment
+              / singleRefSegment / nullableRefSegment / collectionRefSegment "/" keySegment
+
+lastSegment = singleValuedSegment [ "/" castSegment ] [ RWS singlePathCapabilities ]
+            / nullableValuedSegment [ "/" castSegment ] [ RWS nullablePathCapabilities ]
+            / collectionValuedSegment [ "/" castSegment ] [ RWS collectionPathCapabilities ]
+            / singleRefSegment [ "/" castSegment ] [ RWS singleRefPathCapabilities ]
+            / nullableRefSegment [ "/" castSegment ] [ RWS singleNullableReferenceCapability ]
+            / collectionRefSegment [ "/" castSegment ] [ RWS ( collectionRefPathCapabilities / "/" keySegment RWS nullableRefPathCapabilities )]
+
+serviceOperationPath = "/" singleValuedOperation [ "/" castSegment ] [ RWS singlePathCapabilities]
+                     / "/" nullableValuedOperation [ "/" castSegment ] [ RWS nullablePathCapabilities]
+                     / "/" collectionValuedOperation [ "/" castSegment ] [ RWS collectionPathCapabilities]
+                     / "/" singleRefValuedOperation [ "/" castSegment ] [ RWS singleRefPathCapabilities]
+                     / "/" nullableRefValuedOperation [ "/" castSegment ] [ RWS nullableRefPathCapabilities]
+                     / "/" collectionRefValuedOperation [ "/" castSegment ] [ RWS collectionRefPathCapabilities]
+
+singleValuedSegment = identifier                   ; a single, non-nullable property
+                    / singleValuedOperation
```

Comment:
`singleValuedOperation` is just a synonym for `identifier`, so this rule alternative doesn't add any value. Remove it and the synonym rule, and maybe just extend the comment:

```suggestion
singleValuedSegment = identifier                   ; a single, non-nullable property or single-valued operation
```